src/main/java/frc/robot/subsystems/auto/BargeAlign.java

```diff
@@ -27,17 +28,19 @@ public class BargeAlign extends Command {
           .withDriveRequestType(DriveRequestType.OpenLoopVoltage)
           .withForwardPerspective(ForwardPerspectiveValue.BlueAlliance);
 
-  public static boolean atScoringXPosition(CommandSwerveDrivetrain drivebasesubsystem) {
-    double robotX = drivebasesubsystem.getState().Pose.getX();
+  public static boolean atScoringXPosition(DrivebaseWrapper drivebaseWrapper) {
+    double robotX = drivebaseWrapper.getEstimatedPosition().getX();
```

Comment:
Can drivebaseWrapper be null?  Always good to do a null check before dereferencing a variable.